api/apps/api/src/utils/acl.utils.ts

```diff
@@ -113,6 +115,10 @@ export const mapAclDomainToHttpError = (
       return new ForbiddenException(
         `User with ID: ${options?.userId} is not allowed to perform this action on ${options?.resourceType}.`,
       );
+    case projectIsMissingInfoForRegularPus:
+      return new BadRequestException(
+        `Project with regular planning unit shape is missing GADM id or country id.`,
```

Comment:
I suggest to use the descriptive error message from my ill-fated decorator-based PR:
```suggestion
        `When a regular planning grid is requested (hexagon or square) either a custom planning area or a GADM area gid must be provided.`,
```

(needs to be adjusted in tests as well)